PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,12 @@ Fixes #(issue).
 
 # Done is when (DoD):
 - [ ] I have added UI/Unit tests for my changes
+- [ ] Old tests and eslint rule are not broken
```

Comment:
This is already enforced by automation.

Reply:
Still good to have i guess

Reply:
This is not a new rule. i just put it up